docs/data/material/components/about-the-lab/about-the-lab.md

```diff
@@ -19,24 +19,18 @@ For a component to be ready to move to the core, the following criteria are cons
 
 To install and save in your `package.json` dependencies, run one of the following commands:
 
-:::info
-The `next` tag is used to download the latest <b>pre-release</b>, v6 version. Remove it to get the current stable version.
-:::
-
-<!-- #default-branch-switch -->
```

Comment:
Lets keep this one (same as https://github.com/mui/material-ui/pull/43454/files#r1731548827)